sdk/communication/Azure.Communication.Administration/tests/samples/Sample1_CommunicationIdentityClient.cs

```diff
@@ -17,8 +17,14 @@ namespace Azure.Communication.Administration.Samples
     /// </summary>
     public partial class Sample1_CommunicationIdentityClient : CommunicationIdentityClientLiveTestBase
     {
-        public Sample1_CommunicationIdentityClient(bool isAsync): base(isAsync)
-            => Matcher.ExcludeHeaders.Add("x-ms-content-sha256");
+        public Sample1_CommunicationIdentityClient(bool isAsync) : base(isAsync)
+        {
+            Matcher.IgnoredHeaders.Add("x-ms-content-sha256");
+            Matcher.LegacyExcludedHeaders.Add("x-ms-client-request-id");
+            Matcher.LegacyExcludedHeaders.Add("User-Agent");
+            Matcher.LegacyExcludedHeaders.Add("traceparent");
+            Matcher.LegacyExcludedHeaders.Add("Date");
```

Comment:
It looks like the recordings here were relying on these headers being present, but the requests were not including them. Previously this worked because these were defaults of ExcludeHeaders.

Communication friends - should we even have recordings for samples? In general we don't do that. If we want these recordings, we may be able to just re-record rather than adding these extra exclusions. Please let me know what you think.

Reply:
Please disregard the point about needing to add the LegacyExcludedHeaders - I had a bug in the RecordMatcher. The comment about the Samples not typically being recorded is true though.